types/redux-form/redux-form-tests.tsx

```diff
@@ -103,11 +103,11 @@ const MyField: StatelessComponent<MyFieldProps> = ({
     input.onChange("value");
     input.onChange({} as React.SyntheticEvent<HTMLDivElement>);
 
-    input.onDragStart({} as React.DragEvent<HTMLDivElement>);
+    input.onDragStart({} as React.DragEvent<HTMLDivElement>, "newValue", "previousValue");
```

Comment:
This code is wrong. You should be changing the declared types of these properties instead of changing the definition of `EventHandler`